### ISSUE TITLE
Crate version on crates.io is still 0.1

### ISSUE THREAD

**Issue by testforvln** [dovahcrow/treerite#10]
Would you please release the 0.2 version of treerite? Thanks.

**Comment by dovahcrow** [comment]
Yeah I'd like to but am too busy these days to merge several PRs and fix issues for this repo. Might be able to do that later this month.
